### PR TITLE
Update to fictional version :)

### PR DESCRIPTION
**Pull request by ekarlso**
This is a DEMO, dont care about it.

### CODE REVIEW

**Review by pazkooda — CHANGES_REQUESTED** [review]
Consider using actual existing version.

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- runtime/test dependencies -->
-        <quarkus.version>1.13.2.Final</quarkus.version>
+        <quarkus.version>1.13.6.Final</quarkus.version>
```
Comment (pazkooda) [inline pom.xml:22]: Consider using version that acutaly exist.

**Review by pazkooda — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (ekarlso) [reply pom.xml:22]: OK.

Comment (pazkooda) [inline pom.xml:22]:
```suggestion
        <quarkus.version>1.13.2.Final</quarkus.version>
```

**Pull request closed without merging**